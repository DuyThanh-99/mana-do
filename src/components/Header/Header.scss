.header {
  margin-top: 6rem;
  display: flex;
  align-items: center;
  h1 {
    color: #395ad2;
    margin-left: 1rem;
  }
  img {
    width: 60px;
    height: 60px;
    border-radius: 50%;
  }
}
