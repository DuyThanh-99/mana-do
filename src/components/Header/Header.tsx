import React from "react";
import { ManabieLogo } from "../../models/images";
import "./Header.scss";

export const Header: React.FC<IHeader> = ({ title }) => {
  return (
    <div className="header">
      <img src={ManabieLogo} alt="m-logo" />
      <h1 data-testid="Header__title">{title}</h1>
    </div>
  );
};
