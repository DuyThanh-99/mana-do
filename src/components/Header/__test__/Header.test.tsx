import React from "react";
import { render, screen } from "@testing-library/react";
import { Header } from "../Header";

// render
const renderComponent = () => {
  return render(<Header title="MANA-DO" />);
};

// test case
describe("<Header/>", () => {
  it("should match snapshot", () => {
    const wrapper = renderComponent();
    expect(wrapper.container).toMatchSnapshot();
  });

  it("should render UI correctly", () => {
    renderComponent();
    const title = screen.getByText("MANA-DO");
    expect(title).toBeInTheDocument();
  });
});
