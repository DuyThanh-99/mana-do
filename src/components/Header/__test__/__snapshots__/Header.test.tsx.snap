// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> should match snapshot 1`] = `
<div>
  <div
    class="header"
  >
    <img
      alt="m-logo"
      src="m-logo.jpg"
    />
    <h1
      data-testid="Header__title"
    >
      MANA-DO
    </h1>
  </div>
</div>
`;
